pkg/constants/constants.go

```diff
@@ -121,6 +121,13 @@ const (
 	InferenceServiceDefaultLoggerPort = "8081"
 )
 
+// Labels to put on kservice
+const (
+	KServiceComponentLabel = "component"
+	KServiceModelLabel     = "model"
```

Comment:
What is model in this case? Is this effectively the InferenceService's name?

Reply:
Yes the name of the InferenceService. I’ve tried to explain at https://github.com/kubeflow/kfserving/issues/697 but I think I could’ve explained better. I will add an example.